deploy/helm/secret-operator/values.yaml

```diff
@@ -73,7 +59,8 @@ podAnnotations: {}
 labels:
   stackable.tech/vendor: Stackable
 
-podSecurityContext: {}
+podSecurityContext:
+  {}
```

Comment:
```suggestion
podSecurityContext: {}
```

Reply:
Ah goddamn. Need to see if I can fix that at some point...